backend/metadata/databases/default/tables/core_sections.yaml

```diff
@@ -47,6 +47,16 @@ select_permissions:
                 user:
                   user_id:
                     _eq: X-Hasura-User-Id
+  - role: moderator
+    permission:
+      columns:
+        - agreement_id
```

Comment:
```suggestion
```

Explanation: we removed agreement↔️sections relation. We can see the sections of a given agreement using agreement↔️chapters↔️sections

Reply:
done